src/core/mac/mac.hpp

```diff
@@ -829,7 +890,10 @@ class Mac : public InstanceLocator, private NonCopyable
     uint16_t mCslPeriod;
 #endif
     uint8_t mWakeupChannel;
-
+#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
+    uint32_t mWedListenInterval;
+    uint16_t mWedListenDuration;
```

Comment:
The `SubMac` has defined these two variables, could we re-use the `SubMac` defined variables?

Reply:
We keep a copy of the variables in both layers for different usages. Similar situation happens with `mCslPeriod`.